src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java

```diff
@@ -98,8 +98,8 @@ public void anonCanLoginViaUserLoginPage() {
     Assertion.assertTrue(special.isStringInURL(URLsContent.SPECIAL_UPLOAD));
   }
 
-  @Test(groups = "ForcedLogin_004_notLoggedIn")
-  public void anonCanLoginOnSpecialWatchListPage() {
+  @Test(groups = "Login_anonCanLogInOnSpecialWatchListPage")
+  public void anonCanLogInOnSpecialWatchListPage() {
```

Comment:
Login before the underscore is a noun. It's the part of prefix that descrives the tested functionality.
Log in (LogIn) as two separate words is a verb and belongs to a sentence that describes the test case.